jetty-core/jetty-server/src/main/config/modules/security.mod

```diff
@@ -0,0 +1,10 @@
+# DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
```

Comment:
I think both `security.mod` and `jaas.mod` should be in the `jetty-security` maven project.

Reply:
agree, but I've got to warm up my maven-foo to add a config component.   Will do, but will resolve other issues first....